selfdrive/ui/qt/offroad/driverview.cc

```diff
@@ -12,7 +12,7 @@ DriverViewWindow::DriverViewWindow(QWidget* parent) : QWidget(parent) {
   layout = new QStackedLayout(this);
   layout->setStackingMode(QStackedLayout::StackAll);
 
-  cameraView = new CameraViewWidget("camerad", VISION_STREAM_RGB_FRONT, true, this);
+  cameraView = new CameraViewWidget("camerad", VISION_STREAM_YUV_FRONT, true, this);
```

Comment:
wanna make a PR to rename the YUV streams? something like VISION_STREAM_YUV_FRONT -> VISION_STREAM_ROAD and leave RGB as is